doc/releases/changelog-dev.md

```diff
@@ -91,6 +91,9 @@
 
 <h3>Documentation 📝</h3>
 
+* Corrected some spellings and typos scattered throughout the docstrings.
+  [(#4487)](https://github.com/PennyLaneAI/pennylane/pull/4487)
+
```

Comment:
@Anderac, let us know if you'd like your name added to the list of contributors. I'm planning to merge this PR next week.